<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>시뮬레이션 환경 설정</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>321</width>
      <height>331</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>car</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>속도(단위:m/s)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSpinBox" name="spinBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>차종</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="listWidget_Test">
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>경차</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>소형차</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>중형차</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>대형차</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>카메라구도</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QRadioButton" name="groupBox_rad3_10">
         <property name="text">
          <string>전방</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>pedestrian</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>속도(단위:m/s)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_18">
       <item>
        <widget class="QSpinBox" name="spinBox_16"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>카메라구도</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QRadioButton" name="groupBox_rad3_9">
         <property name="text">
          <string>전방</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>키(단위::cm)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QSpinBox" name="spinBox_13"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>drone</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>속도(단위:m/s)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QSpinBox" name="spinBox_11"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>고도(단위: m미터)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QSpinBox" name="spinBox_7"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>191</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>기종</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QListWidget" name="listWidget_Test_2">
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>고정익(Fixed Wing)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>회전익(Rotary Wing)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>동축반전형(Co-axual)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>다중 로터형</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>틸트로터형(Tilt-Rotor)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_12">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>카메라 구도</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QRadioButton" name="groupBox_rad3_8">
         <property name="text">
          <string>전방</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>airplane</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>고도(단위: m미터)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_23">
       <item>
        <widget class="QSpinBox" name="spinBox_21">
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_17">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>카메라 구도</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_24">
       <item>
        <widget class="QRadioButton" name="groupBox_rad3_6">
         <property name="text">
          <string>전방</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="groupBox_rad3_7">
         <property name="text">
          <string>아래고정</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>기종</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_25">
       <item>
        <widget class="QListWidget" name="listWidget_Test_4">
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>비행기</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>글라이더</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>헬리콥터</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>제트기</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>전투기</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>여객기</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>정찰기</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>속도(단위:m/s)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_26">
       <item>
        <widget class="QSpinBox" name="spinBox_24"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>131</width>
      <height>158</height>
     </rect>
    </property>
    <property name="title">
     <string>날씨 설정</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="groupBox_rad1">
       <property name="text">
        <string>맑음</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad3">
       <property name="text">
        <string>흐림</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad4">
       <property name="text">
        <string>비</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad4_2">
       <property name="text">
        <string>눈</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad4_3">
       <property name="text">
        <string>미세먼지</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>환경설정</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="groupBox_rad_park">
       <property name="text">
        <string>AbandonedPark</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad_africa">
       <property name="text">
        <string>Africa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad_block">
       <property name="text">
        <string>Blocks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad_nh">
       <property name="text">
        <string>AirSimNH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad_building">
       <property name="text">
        <string>Building99</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad_mt">
       <property name="text">
        <string>LandscapeMountains</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad_msb">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>MSBuild2018</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad_trap">
       <property name="text">
        <string>TrapCamera</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="groupBox_rad_zhang">
       <property name="text">
        <string>ZhangJiajie</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>실행</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
